docs/source/developer_documentation/development_tutorial/creating_a_report.md

```diff
@@ -50,7 +50,7 @@ class GreetingsReport(Report):
     id = "greetings-report"
     name = _("Greetings report")
     description = _("Makes a nice report about the selected greeting objects")
-    plugins = {"required": [], "optional": []}
+    plugins = {"required": [], "optional": {}}
```

Comment:
Should both be a set?

Reply:
Ah yes forgot to update the docs after realizing this for the definitions